validate.py

```diff
@@ -17,9 +18,11 @@ def user(self, user):
         uValid = re.compile(ur'^[A-Za-z0-9.-_]{1,30}$')
         if re.search(uValid, user):
             self.logger.debug("User %s passed regex check", user)
+            for word in self.blacklist:
+                if (word in user.lower()):
```

Comment:
this isn't a good check, what if I wanted the username "IAmGroot"?  Because it contains 'root' it would fail your blacklist.